code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -105,10 +105,6 @@
 
 		update_muts=1
 
-	if(type == /mob/living/carbon/monkey)
```

Comment:
>Don't change code without testing it kids

I just used the "Check Known Languages" verb as a monkey on a freshly updated local server.
Monkeys don't seem to have any known languages.